guardrails/validators.py

```diff
@@ -1974,16 +1975,29 @@ def __init__(
             on_fail,
             validation_method=validation_method,
             llm_callable=llm_callable,
+            openai_api_key=openai_api_key,
```

Comment:
Do they want to specifically set this in the XML?
They can already pass it in to the Guard on call or parse via a kwarg and it will be propagated to the LLM

Reply:
Let's sync after standup to discuss more on this?